Services/ConfigSettingsService.cs

```diff
@@ -239,17 +248,56 @@ public bool BatVision
 		}
 	}
 
+	/// <summary>
+	/// Adds/updates record in <see cref="BloodBound"/>.
+	/// Saves current config to a file.
+	/// </summary>
+	/// <param name="id">Record key.</param>
+	/// <param name="value">Record value.</param>
+	public void SetBloodBound(PrefabGUID id, bool value)
+	{
+		if (!Core.PrefabCollectionSystem.PrefabGuidToNameDictionary.TryGetValue(id, out string key))
+		{
+			key = id.GuidHash.ToString();
+		}
+
+		config.BloodBound[key] = value;
+		SaveConfig();
+	}
+
+	/// <summary>
+	/// Removes records for <see cref="BloodBound"/>.
+	/// Saves current config to a file.
+	/// </summary>
+	/// <param name="keys">Removes records from</param>
+	public void ClearBloodBound(IEnumerable<string> keys)
+	{
+		foreach (var key in keys)
+		{
+			config.BloodBound.Remove(key);
+		}
+
+		SaveConfig();
+	}
+
 	struct Config
 	{
 		public Config()
 		{
+			BloodBound = new Dictionary<string, bool>();
```

Comment:
The initialization can be simplified.

```suggestion
			BloodBound = [];
```